index.js

```diff
@@ -1,4 +1,4 @@
-var	sanitizeHtml = require('sanitize-html'),
+var	sanitizeHtml = require('/Users/ryan/Verys/sanitize-html'),
```

Comment:
that looks like a local module, that won't work

Reply:
Doh! Updated :).